src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -45,24 +46,23 @@ public GitHubPackage(IServiceProvider serviceProvider)
             this.serviceProvider = serviceProvider;
         }
 
-        protected override void Initialize()
+        protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            base.Initialize();
-            IncrementLaunchCount();
+            var usageTracker = await GetServiceAsync(typeof(IUsageTracker)) as IUsageTracker;
+            await tasks.Task.Run(() => usageTracker.IncrementLaunchCount());
```

Comment:
@AArnott @BertanAygun I was just testing and got a deadlock on the `InitializeAsync` method doing a `GetServiceAsync` call, and the `InitializeAsync` method is definitely getting called on the main thread (see screenshot). The package is to `AllowsBackgroundLoading = true`.

![screen shot 2016-11-02 at 1 20 43 pm](https://cloud.githubusercontent.com/assets/310137/19928736/246f4e62-a100-11e6-81be-1f66bee26cbd.png)